### ISSUE TITLE
msys2/mingw64 python3 compile fails like it's not finding python.h

### ISSUE THREAD

**Issue by sanderboer** [tpaviot/pythonocc-core#47]
Hi,
I can report I am happily coding away in python2 with pythonocc, but I wanted to get with the times and get it to work with python3.
```
CMakeFiles/_Adaptor2d.dir/objects.a(Adaptor2dPYTHON_wrap.cxx.obj):
Adaptor2dPYTHON_wrap.cxx:(.text+0x15f51): undefined reference to `__imp__Py_NoneStruct'
C:/msys64/mingw64/lib/gcc/x86_64-w64-mingw32/4.9.2/../../../../x86_64-w64-mingw32/bin/ld.exe: 
CMakeFiles/_Adaptor2d.dir/objects.a(Adaptor2dPYTHON_wrap.cxx.obj):  
bad reloc address 0x100 in section `.data'
collect2.exe: error: ld returned 1 exit status
CMakeFiles/_Adaptor2d.dir/build.make:93: recipe for target 'win64/bin/RELEASE/_Adaptor2d.pyd' failed
make[2]: *** [win64/bin/RELEASE/_Adaptor2d.pyd] Error 1
```

I reported the reloc address error before and Jelle suggested it is because PYTHON_INCLUDE_DIR is not set properly. This time it is set properly, but alas.

During cmake I see this :
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: unorderable types: NoneType() >= tuple()
```
It is quite persistent and no clue where that is coming from. I guessed CMAKE_SWIG_FLAGS where not set to "-py3" so I tried setting manually to no avail.

tips ?

gr.Sander

**Comment by tpaviot** [comment]
Lastest pythonocc relase (0.16.2) includes Mingw64 compilation fixes.

**Comment by monkpit** [comment]
What was the fix? I am using 16.2 with mingw and I'm getting the unorderable types error.
